speech_assets/SampleUtterances.txt

```diff
@@ -324,16 +324,12 @@ Mute toggle unmute
 Mute unmute
 NewShowInquiry if there are any episodes of {Show}
```

Comment:
It'll happen. Just trying to establish a baseline and push up as few changes as possible to get this branch mergeable.